services/integration/src/controllers/constitution.controllers/constitution.js

```diff
@@ -1,7 +1,16 @@
-const Constitution = require('../models/constitution.model');
-const { asyncHandler } = require('../utils');
+const Constitution = require('../../models/constitution.model');
+const { asyncHandler } = require('../../utils');
 
 const create = asyncHandler(async (req, res) => {
+
+  const actOfConstitution = new Constitution(req.body);
```

Comment:
this const should be named as just `constitution`

Reply:
ok , I rename the const for constitution

Reply:
I have changed the name